frontend/src/js/components/leftnav.js

```diff
@@ -35,6 +35,10 @@ const listItems = [
   { route: '/deployments', text: 'Deployments', canAccess: ({ userCapabilities: { canDeploy, canReadDeployments } }) => canReadDeployments || canDeploy },
   { route: '/auditlog', text: 'Audit log', canAccess: ({ userCapabilities: { canAuditlog } }) => canAuditlog }
 ];
+const spTenantItems = [
+  { route: '/tenants', text: 'Tenants', canAccess },
+  { route: '/auditlog', text: 'Audit log', canAccess }
+];
```

Comment:
these should go into the regular `listItems` and have their access checks adjusted